tools/coqdoc/coqdoc.css

```diff
@@ -327,3 +327,8 @@ ul.doclist {
     margin-top: 0em;
     margin-bottom: 0em;
 }
+
+:target {
```

Comment:
```suggestion
.code :target {
```

To restrict the rule to code blocks. I'm not sure the same rule would work elsewhere.

Reply:
Yes, much better.